be/src/exprs/expr.cpp

```diff
@@ -283,6 +283,9 @@ Status Expr::create_tree_from_thrift(ObjectPool* pool, const std::vector<TExprNo
     Expr* expr = NULL;
     RETURN_IF_ERROR(create_expr(pool, nodes[*node_idx], &expr));
     DCHECK(expr != NULL);
```

Comment:
Here is the DCHECK here. So I think if expr is null here, this is a fatal error?
Could you show me the case which can result in expr as null there?

Reply:
some bad cases which lead to Doris-BE process fail in meituan are fixed by [https://github.com/apache/incubator-doris/pull/5799](https://github.com/apache/incubator-doris/pull/5799).
so this pr can be closed.